test/custom-elements/index.js

```diff
@@ -45,7 +45,7 @@ describe('custom-elements', function() {
 	before(async () => {
 		svelte = loadSvelte();
 		server = await create_server();
-		browser = await puppeteer.launch();
+		browser = await puppeteer.launch({args: ['--no-sandbox', '--disable-setuid-sandbox']});
```

Comment:
Is this specific to the Docker setup or did you find a problem with launching the `custom-elements` scripts?
I ask because every so often, the scripts in this category fail on a hung process.

Reply:
Yes, this is specific for dockerized environments `--no-sandbox` is required.